services/ui-src/src/styles/_main.scss

```
@@ -185,6 +185,44 @@ img {
     }
   }
 
+  .sortable-table {
+
+    .cell-bolded {
+      color: #333;
+      font-weight: 700;
+    }
+
+    thead {
+      display: none;
+    }
+
+    .cell-header {
+      font-weight: 700;
+    }
+
+    @media only screen and (min-width: 768px) {
+      thead {
+        display: table-header-group;
+      }
+
+      .cell-header {
+        display: none;
+      }
+    }
+
+    @media only screen and (max-width: 767px) {
+      tr {
+        display: grid;
+        padding: 12px 0;
+      }
+
+      td {
+        width: 100%;
+        padding: 4px 16px;
+      }
+    }
+  }
```

Nitpick: I think this same code can be written with only one media query instead of two. Plus I move all class definitions above element definitions.

Goal: improve the readability for later troubleshoots on how would someone compare mobile styles versus desktop styles.


```suggestion
    /**
    * Responsive breakpoints
    * 
    * ```
    * 0px          768px
    * |-------------|-------------->
    * |-Mobile------|-Desktop------>
    * ```
    */
    .sortable-table {
    .cell-bolded {
        color: #333;
        font-weight: 700;
    }

    .cell-header {
        font-weight: 700;
    }

    thead {
        display: none;
    }

    tr {
        display: grid;
        padding: 12px 0;
    }

    td {
        width: 100%;
        padding: 4px 16px;
    }

    @media only screen and (min-width: 768px) {
        .cell-header {
          display: none;
        }

        thead {
          display: table-header-group;
        }

        tr {
            display: table-row;
            padding: 0;
        }

        td {
            width: initial;
            padding: 16px;
        }
      }
    }

```